src/pages/en/design-accessible-services/designing-for-users-of-screen-readers.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Designing for users of screen readers
+layout: layouts/base.njk
+description: description
+tags: designingAccessible
+toogle: concevoir-pour-les-utilisateurs-de-revues-decran
+---
+<p>Printable posters <span id="das1">(PDF format)</span>:</p>
+<ul>
+		<li><a href="{{ rootPath }}docs/posters/ScreenReader-en_2023.pdf" id="das8" aria-labelledby="das8 das1">Designing for users of screen readers</a></li></ul>
+
+# Designing for users of screen readers
```

Comment:
Remove the `h1`

Reply:
I have made the required changes; take a look and let me know if there are any more changes needed.

Reply:
I have made the required changes; take a look and let me know if there are any more changes needed.